src/workerd/api/http.c++

```diff
@@ -1136,6 +1136,21 @@ jsg::Ref<Request> Request::constructor(
     headers = js.alloc<Headers>();
   }
 
+  // If the request has a body from a JS stream with expectedLength and doesn't specify
+  // "Content-Length" or "Transfer-Encoding", set "Content-Length" to the expectedLength.
+  // This ensures handlers for requests can access known body lengths from JS streams.
```

Comment:
Sorry to be that guy but, even though it is a bug fix, could it break someone? 
Or in other words, looks to me like we need a compat flag, convince me otherwise 😅

Reply:
Once I think about this one more time, I think you're right. This is technically a breaking change.